index.js

```diff
@@ -93,7 +93,7 @@ var MysqlDriver = Base.extend({
     );
     return {
       foreignKey: constraint.foreignKey,
-      constraints: [escapedName, t, len, constraint.constraints].join(' ')
+      constraints: [escapedName, t, len, constraint.constraints].join(' '),
```

Comment:
please comfort your auto fixer with db-migrates internal rules:

https://github.com/db-migrate/mysql/blob/master/.prettierrc
https://github.com/db-migrate/mysql/blob/master/.eslintrc

trailing commas are not part of this for example.